components/automate-ui/src/app/pages/project/list/project-list.component.ts

```diff
@@ -262,7 +262,9 @@ export class ProjectListComponent implements OnInit, OnDestroy {
   public getProjectStatus(project: Project): string {
     const cachedStatus = this.statusCache[project.id];
     let result: string;
-    if (this.applyRulesInProgress) {
+    if (project.status === 'NO_RULES') {
+      result = 'OK';
```

Comment:
Unrelated to this work, but is it weird for this to say `OK` if there are no rules-- _all_ projects are updated during an update, even if they don't have rules, correct?

Reply:
It is actually rules that are being applied during a project update so if no rules, nothing changes on the project.

Reply:
That's what it sort of seems like to the user, but from our angle, every project will be updated, bc it's _never_ the project being updated during this call, it's the nodes 🤷‍♂ . I think it's confusing.